smpte.js

```diff
@@ -632,44 +637,80 @@ function insertConformance(docMetadata) {
 
     } else {
 
-      implConformance = sec.innerText.innerHTML;
+      implConformance = sec.innerText;
```

Comment:
```suggestion
      implConformance = sec.innerHTML;
```

Reply:
Updated in https://github.com/SMPTE/html-pub/pull/287/commits/212bd78f67077d21a0bbed3745d62b46f3ada8ab